hack/lib/golang.sh

```diff
@@ -735,27 +735,27 @@ kube::golang::build_binaries() {
       for platform in "${platforms[@]}"; do (
           kube::golang::set_platform_envs "${platform}"
           kube::log::status "${platform}: build started"
-          kube::golang::build_binaries_for_platform ${platform}
+          kube::golang::build_binaries_for_platform "${platform}"
           kube::log::status "${platform}: build finished"
         ) &> "/tmp//${platform//\//_}.build" &
       done
 
       local fails=0
-      for job in $(jobs -p); do
-        wait ${job} || let "fails+=1"
+      for job in "$(jobs -p)"; do
```

Comment:
I think this is the source of the current breakage - `jobs -p` returns a whitespace-separated list of PIDs, but encapsulating it in quotes means the for loop is treating them all as one unit.

Reply:
i.e. exactly the same thing as in https://github.com/kubernetes/kubernetes/pull/72933#issuecomment-454936261

Reply:
(in both cases I think it's correct to unquote the variable)